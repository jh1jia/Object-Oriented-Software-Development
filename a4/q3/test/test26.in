test22.txt dropfirst 1 count D count c count 0 count 5 count H allcaps
