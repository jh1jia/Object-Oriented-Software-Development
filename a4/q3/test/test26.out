123DDDD
B1DEFG1
123AA23
1%DC
C
3
4D22#DCE34
2
